framework/src/modules/interoperability/mainchain/endpoint.ts

```diff
@@ -13,10 +13,35 @@
  */
 
 import { BaseInteroperabilityEndpoint } from '../base_interoperability_endpoint';
-import { CHAIN_REGISTRATION_FEE } from '../constants';
+import { CHAIN_REGISTRATION_FEE, MIN_RETURN_FEE_PER_BYTE_LSK } from '../constants';
+import { getCCMSize } from '../utils';
+import {
+	CROSS_CHAIN_COMMAND_NAME_TRANSFER,
+	USER_SUBSTORE_INITIALIZATION_FEE,
+} from '../../token/constants';
+import { ModuleEndpointContext } from '../../../types';
+import { CCMsg } from '../types';
 
 export class MainchainInteroperabilityEndpoint extends BaseInteroperabilityEndpoint {
 	public getRegistrationFee(): { fee: string } {
 		return { fee: CHAIN_REGISTRATION_FEE.toString() };
 	}
+
+	public getMinimumMessageFee(context: ModuleEndpointContext): { fee: string } {
```

Comment:
probably it's enough to just return the `MIN_RETURN_FEE_PER_BYTE` instead of computing